src/hotspot/share/opto/compile.cpp

```diff
@@ -719,6 +719,11 @@ Compile::Compile( ciEnv* ci_env, ciMethod* target, int osr_bci,
     method()->ensure_method_data();
   }
 
+  if (StressLCM || StressGCM || StressIGVN || StressCCP ||
```

Comment:
We need to initialize the seed earlier because `StressUnstableIfTraps` is already used during parsing.